### ISSUE TITLE
Spires without lightning rods still don't fully protect area around them

### ISSUE THREAD

**Issue by aquila-regalis** [BTW-Community/BTW-Public#373]
Running CE 2.1.1, this building got struck by lightning:
![2023-09-17_16 57 23](https://github.com/BTW-Community/BTW-Public/assets/145294821/99f892dc-c60e-4408-856e-6a8430507d65)
(black line of smoldering logs near the left corner is where the lightning hit)
Based on my understanding of the lightning algorithm my guess is on the first pass it jumped to one of the outer pillars and then only went to the building on the second pass. The setup here is somewhat artificial but something similar could happen in survival if the nearby terrain isn't flat.

**Comment by LannyRipple** [comment]
Could be wrong about this and strongly informed by how I first learned lightning works.  I think you might have built _just_ the correct setup that lightning avoids the outer pillars and central pillar.  This would set up a small square in your NW (N being upper left for this example) quadrant that would not find a pillar before it found the corner of your wood structure.

I saw Dawnraider's comments that this shouldn't be the case from recent changes but I peeked at the code and boundaries still seem to be an 8x8 box unless there is a lightning rod (17x17) involved.  You might try pulling in your outer pillars until you find (above) doesn't happen (RL and time permitting) and adding that to the bug report.

**Comment by ammoore00** [comment]
I don't think that would add any useful information tbh, I already know exactly what the issue is, it's just figuring out an actual solution that fixes it, if it is even fixable.
